docs/detections/alerts-view-details.asciidoc

```diff
@@ -83,12 +101,12 @@ When gathering threat intelligence data for an alert, {elastic-sec} queries the
 
 If these fields aren't available, {elastic-sec} does not perform the query and does not display threat intelligence data for the alert.
 
-Discovered threat indicator match data is placed under the **Threat Match Detected** subsection of the Overview tab.
+Available threat indicator match data is placed under the **Threat Match Detected** subsection of the Enriched data section.
```

Comment:
Prob not necessary to specify all the levels and sublevels, we've already established that we're talking about the main "Enriched data" section
```suggestion
Available threat indicator match data appears in the **Threat Match Detected** subsection.
```